frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove default Readme generated by create-react-app and add your project component details, figma file etc.